parmed/gromacs/gromacstop.py

```diff
@@ -1422,6 +1423,20 @@ def write(self, dest, combine=None, parameters='inline'):
                               atom_type.mass, atom_type.charge, atom_type.sigma/10,
                               atom_type.epsilon*econv))
             parfile.write('\n')
+            # Nonbonded parameters
+            if self.has_NBFIX():
+                typemap = dict(self.parameterset.nbfix_types)
+                dest.write('[ nonbond_params ]\n')
+                eps_conversion = u.kilocalorie.conversion_factor_to(u.kilojoule)
+                for key, val in typemap.items():
+                    a1 = key[0]
+                    a2 = key[1]
+                    eps = val[0] # kcal
+                    sig = val[1] # Angstrom
+                    eps *= eps_conversion
+                    sig *= 0.1
+                    dest.write('{0} {1} 1 {2} {3}\n'.format(
+                        key[0], key[1], sig/2**(1/6), eps))
```

Comment:
Looks like key[0] and key[1] can be replaced with a1 and a2 respectively. Or just get rid of a1 and a2. 

These unused variables are breaking the build because there’s a linting gate that fails the tests when errors are detected.